Bottom: 9528eed47ccf49407bcd09869b030b3a0de4d2b8
Top:    7803913a89226d9fcb0ed108695a9460d8e2d5ce
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-20 23:18:17 +0000

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!










---

diff --git a/src/gpcb-menu.res.in b/src/gpcb-menu.res.in
index 598324e..a4f9c3d 100644
--- a/src/gpcb-menu.res.in
+++ b/src/gpcb-menu.res.in
@@ -333,13 +333,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
